h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -53,9 +54,16 @@ function AnnotationListContent({
         className="border rounded p-2 text-center"
         data-testid="annotations-fallback-message"
       >
-        {filterStatus === 'PENDING'
-          ? 'You are all set!'
-          : 'No annotations found for selected status'}
+        {!filterStatus && 'There are no annotations in this group.'}
+        {filterStatus && (
+          <>
+            There are no{' '}
+            <span className="font-bold lowercase">
+              {moderationStatusToLabel[filterStatus]}
+            </span>{' '}
```

Comment:
I think there was a mention in Slack of using regular rather than bold weight here? I think I'm inclined to prefer regular weight.